SmartDeviceLink/SDLSystemCapability.m

```diff
@@ -82,6 +82,18 @@ - (instancetype)initWithRemoteControlCapability:(SDLRemoteControlCapabilities *)
     return self;
 }
 
+- (instancetype)initWithDisplayCapabilities:(NSArray<SDLDisplayCapability *> *)capabilities {
+    self = [self init];
+    if (!self) {
+        return nil;
+    }
+    
+    self.systemCapabilityType = SDLSystemCapabilityTypeDisplays;
+    self.displayCapabilities = [capabilities copy];
```

Comment:
No need to copy this because it's an immutable array.

Reply:
The parameter type is NSArray but the object could be the subclass NSMutableArray.